include/mgba/internal/gba/hardware-extensions.h

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) 2021 Jeffrey Pfau
```

Comment:
```suggestion
/* Copyright (c) 2013-2021 Jeffrey Pfau
```

This file should probably be put in `extra` and named something shorter, like `extensions`.

Reply:
Fixed on all three files.